testcases/ooxml/XSSF/Streaming/SheetDataWriterTests.cs

```diff
@@ -65,16 +65,17 @@ public void IfCallingEmptyConstructorShouldCreateNonZippedTempFileNonDecoratedSt
         public void IfWritingRowWithCustomHeightShouldIncludeCustomHeightXml()
         {
             _objectToTest = new SheetDataWriter();
-            var row = new SXSSFRow(null);
-            row.Height = 1;
+            var row = new SXSSFRow(null) {
+                Height = 1
+            };
 
             _objectToTest.WriteRow(0, row);
             _objectToTest.Close();
 
             var lines = File.ReadAllLines(_objectToTest.TemporaryFilePath());
 
             Assert.True(lines.Length == 2);
-            Assert.AreEqual("<row r=\"" + 1 + "\" customHeight=\"true\" ht=\"" + row.HeightInPoints + "\">", lines[0]);
+            Assert.AreEqual("<row r=\"" + 1 + "\" customHeight=\"1\" ht=\"" + row.HeightInPoints.ToString().Replace(',', '.') + "\">", lines[0]);
```

Comment:
Will it be better to use `InvariantCulture` instead of comma to point replacement?